### PR TITLE
Update UG - image table

### LINKED ISSUE #231: [PE-D] Cannot edit images of recipes
Steps to reproduce:
1. Edit any recipe with new image

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: todoge/ped#12

### PR DESCRIPTION
**Pull request by jiaqi20**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -216,19 +216,30 @@ Multiple ingredients are separated by `,`.
...
+* `IMAGE` can be in two formats:
+    * Local path e.g. images/healthy1.jpg 
+    * URL (online image) e.g. https://vaya.in/recipes/wp-content/uploads/2018/06/Club-sandwich.jpg
+
+* Here are the respective actions required by users depending on different usage of `IMAGE`:
+
+  Usage | Action | Example | Outcome
+  ---------|-----------------------|---------|---------
+  Image from local storage | 1. Specify **absolute path** of the image<br><br> 2. Add **file://** in front of the file path<br><br> 3. Replace **IMAGE** in img/IMAGE with the file path of your image | file:///D:/images/wishful<br>/data/myimage.png | Local image will be displayed
+  Image from online resources | 1. Make sure the computer is connected to Internet<br><br> 2. Copy the online **image address** | Valid image address which starts with https:// and ends with .jpg/jpeg/png | The image will be downloaded into data folder and displayed
```
Comment (TianYong-Goh) [inline docs/UserGuide.md]: I feel the.jpg/jpeg/png a bit confusing people might not take the '/' as 'or'.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -216,19 +216,30 @@ Multiple ingredients are separated by `,`.
...
+  Sample images | Replace IMAGE in img/IMAGE with the **file path**<br> (refer to table [below](#table) ) | img/images/healthy1.jpg | Sample image is displayed
+  Invalid image | NA | 1. Invalid local file path<br><br>2. Invalid URL<br><br>3. No internet connection | Default image will be displayed
+  No image input | NA | NA | Default image will be displayed since `IMAGE` is optional field
```
Comment (TianYong-Goh) [inline docs/UserGuide.md]: Should we caps and bold the  'Optional Field'? I not sure haha

File: docs/UserGuide.md [hunk 3]
```diff
@@ -296,11 +307,20 @@ Multiple ingredients are separated by `,`.
...
+* `IMAGE` can be in two formats:
+    * Local path e.g. images/healthy1.jpg 
+    * URL (online image) e.g. https://vaya.in/recipes/wp-content/uploads/2018/06/Club-sandwich.jpg
+
+* Here are the respective actions required by users depending on different usage of `IMAGE`:
+
+  Usage | Action | Example | Outcome
+  ---------|-----------------------|---------|---------
+  Image from local storage | 1. Specify **absolute path** of the image<br><br> 2. Add **file://** in front of the file path<br><br> 3. Replace **IMAGE** in img/IMAGE with the file path of your image | file:///D:/images/wishful<br>/data/myimage.png | Local image will be displayed
+  Image from online resources | 1. Make sure the computer is connected to Internet<br><br> 2. Copy the online **image address** | Valid image address which starts with https:// and ends with .jpg/jpeg/png | The image will be downloaded into data folder and displayed
```
Comment (TianYong-Goh) [inline docs/UserGuide.md]: same as above

File: docs/UserGuide.md [hunk 4]
```diff
@@ -296,11 +307,20 @@ Multiple ingredients are separated by `,`.
...
+  Sample images | Replace IMAGE in img/IMAGE with the **file path**<br> (refer to this [table](#table) ) | img/images/healthy1.jpg | Sample image is displayed
+  Invalid image | NA | 1. Invalid local file path<br><br>2. Invalid URL<br><br>3. No internet connection | Default image will be displayed
+  No image input | NA | NA | Default image will be displayed since `IMAGE` is optional field
```
Comment (TianYong-Goh) [inline docs/UserGuide.md]: same as above

File: docs/UserGuide.md (hunk 1 above)
Comment (caitlinjee) [reply docs/UserGuide.md]: agree with tianyongs comment

**Review by caitlinjee — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (jiaqi20) [reply docs/UserGuide.md]: okayy!

File: docs/UserGuide.md (hunk 2 above)
Comment (jiaqi20) [reply docs/UserGuide.md]: yesyes hahaha thanks!

**Review by oliviajohansen — APPROVED** [review]
lgtm

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Review by daongochieu2810 — APPROVED** [review]

**Pull request merged**